### ISSUE TITLE
Move panel transparency setting here

### ISSUE THREAD

**Issue by danrabbit** [elementary/switchboard-plug-pantheon-shell#129]
Following the theme that a11y settings are just settings, perhaps the "Dock" tab should be renamed to something more inclusive and we can add the panel transparency setting here too
